ctfo_server.h

```diff
@@ -1116,27 +1140,88 @@ class CTFOServer final {
   }
 
  private:
+  struct PushNotificationsSender {
+    Storage& storage;
+    const std::chrono::microseconds starting_from;
+    current::integrations::onesignal::IOSPushNotificationsSender transport;
+
+    using transaction_t = typename Storage::transaction_t;
+
+    PushNotificationsSender(Storage& storage,
+                            std::chrono::microseconds last_pushed_notification_timestamp,
+                            const std::string& onesignal_app_id,
+                            uint16_t onesignal_local_port)
+        : storage(storage),
+          starting_from(last_pushed_notification_timestamp + std::chrono::microseconds(1)),
+          transport(onesignal_app_id, onesignal_local_port) {
+      std::cerr << "Starting sending push notifications from " << starting_from.count() << " epoch us.\n";
```

Comment:
It's defined in the outer class. OK to remove altogether, it was here only to debug the logic of when to resume from.